SpatialGDK/Extras/schema/query_tags.schema

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+// System query tags for entity completeness
+
+component ServerAuthTag {
+    id = 1000;
+}
+
+component ServerNonAuthTag {
+    id = 1001;
+}
+
+component ClientAuthTag {
```

Comment:
Thought - should client auth be called ownership? Not sure it matters too much right now.

Reply:
these names are the same as the interest result type names. I like these because the completeness is based on whether the client is auth over the entity. I imagine that means the same thing as owning, but in more of a spatial way where owning is the unreal one. I'm indifferent here. If we are going to change it, now would be a good time

Reply:
Auth seems good to keep abstractions separate. As the non auth form also cares about some ownership stuff too.